packages/loaders/git/src/index.ts

```diff
@@ -34,11 +34,9 @@ export type GitLoaderOptions = SingleFileOptions & {
   /**
    * Additional options to pass to `graphql-tag-pluck`
    */
-  pluckConfig: GraphQLTagPluckOptions;
+  pluckConfig?: GraphQLTagPluckOptions;
```

Comment:
Making this optional since `graphql-tag-pluck` doesn't actually require an options object to be passed in.